core/store/orm/config.go

```diff
@@ -165,19 +165,29 @@ func (c Config) MinimumServiceDuration() time.Duration {
 	return c.viper.GetDuration(EnvVarName("MinimumServiceDuration"))
 }
 
-// EthGasBumpThreshold represents the maximum amount a transaction's ETH amount
-// should be increased in order to facilitate a transaction.
+// EthGasBumpThreshold is the number of blocks to wait for confirmations before bumping gas again
 func (c Config) EthGasBumpThreshold() uint64 {
 	return c.viper.GetUint64(EnvVarName("EthGasBumpThreshold"))
 }
 
-// EthGasBumpWei represents the intervals in which ETH should be increased when
-// doing gas bumping.
+// EthGasBumpPercent is the minimum percentage by which gas is bumped on each transaction attempt
+// Change with care since values below geth's default will fail with "underpriced replacement transaction"
```

Comment:
Maybe add a panic if this is below 10, or maybe less than `go-ethereum/core/DefaultTxPoolConfig.PriceBump` (and if so, a warning that this value might be inaccurate if the price bump was customized at geth/parity startup)?

If we wanted to stick to the linear increase, we could configure this to be 0 at startup via the `txpool.pricebump` flag. That should not get in the way of this PR, though.

Reply:
@coventry OK, not a bad idea. Where are you imagining we would check this - on config read or on application startup?